src/index.html

```diff
@@ -6,6 +6,7 @@
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="global.css">
```

Comment:
I added this global stylesheet because I think the footer will appear in every single page the same. That avoids us having to copy and paste the same code for every single different stylesheet file specific to a page.

Reply:
OK, makes sense to me.